extensions/markdown-language-features/src/docIndex.ts

```diff
@@ -0,0 +1,50 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as vscode from 'vscode';
+import { Disposable } from './util/dispose';
+
+
+export class DocumentIndex extends Disposable {
+	private _uriMap: Map<string, vscode.TextDocument> = new Map();
```

Comment:
[nit] `readonly` and avoid having an explicit type signature where possible

Reply:
Fixed.